bazel/envoy_mobile_dependencies.bzl

```diff
@@ -22,7 +22,7 @@ def kotlin_dependencies():
         artifacts = [
             # Kotlin
             "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.11",
-
+            "androidx.recyclerview:recyclerview:1.1.0",
```

Comment:
If this is only used in the example app, can it be removed from these dependencies?

Reply:
I can pull it to be a parameter if that helps but we'll have to declare it

Reply:
I guess my vague misgiving is that it adds a new external dependency to the build (which could break), which isn't strictly necessary to build the library.

That said, if it's maintained by Google, it's probably fairly safe.

You could move this into the BUILD file of the example app though, couldn't you?

Reply:
(Or to a function that only gets called when building those targets.)

Reply:
We spoke offline. I do agree that we should keep this as minimal as possible. @goaway is fine with making the change to clean this up as a follow up PR

Reply:
https://github.com/lyft/envoy-mobile/issues/950